lib/SyTest/Federation/Client.pm

```diff
@@ -16,8 +16,7 @@ use SyTest::Assertions qw( :all );
 use URI::Escape qw( uri_escape );
 
 use constant SUPPORTED_ROOM_VERSIONS => [qw(
-   1 2 3 4 5
-   org.matrix.msc2260
```

Comment:
This room version was previously removed from Synapse, it also seems a bit weird to bit unstable room versions in here.